packages/components/pages/f-registration/src/components/Registration.vue

```diff
@@ -46,6 +46,7 @@
                     v-model="firstName"
                     name="firstName"
                     required
+                    :isVisuallyRequired=false
```

Comment:
I think you'll need quotes around all of these, and props should be in kebab-case please
```suggestion
                    :is-visually-required="false"
```